src/report/business-reports/payables.scm

```diff
@@ -37,6 +37,7 @@
 
 (define acc-page gnc:pagename-general)
 (define this-acc (N_ "Payable Account"))
+(define receivable #f)		;; receivable = #t, payable = #f
 
 (define (options-generator)    
   (let* ((options (gnc:new-options)) 
```

Comment:
Right below this line in the original code one payables-specific option is added already. It's here you can add your own internal one. You could call it "receivable" and give it a default value #f to get as close as possible to your original intent.